packages/api/src/external/commonwell/patient.ts

```diff
@@ -436,13 +447,54 @@ async function updatePatientAndLinksInCw(
       },
     });
 
-    await queryDocsIfScheduled(patient, getOrgIdExcludeList);
+    let foundNewDemographics = false;
+    if (rerunPdOnNewDemographics) {
+      foundNewDemographics = checkForNewDemographics(patient, cwLinks);
+    }
+
+    let scheduledPdRequest = getCWData(patient.data.externalData)?.scheduledPdRequest;
+    if (foundNewDemographics) {
+      const updatedPatient = await scheduleOrRunPatientDiscovery({
+        patient,
+        facilityId,
+        rerunPdOnNewDemographics: false,
+        augmentDemographics: true,
+        isRerunFromNewDemographics: true,
```

Comment:
Dont we need to also wait until the `statusCw`is complete? If not this will be called right after `scheduleOrRunPatientDiscovery` is set even though it can still be processing